src/features/trustedDevice/pages/SuspensionChoice.native.test.tsx

```diff
@@ -40,11 +71,36 @@ describe('<SuspensionChoice/>', () => {
   })
 
   it('should open CGU url when clicking on "conditions générales d’utilisation" button', () => {
-    render(<SuspensionChoice />)
+    renderSuspensionChoice()
 
     const cguButton = screen.getByText('conditions générales d’utilisation')
     fireEvent.press(cguButton)
 
     expect(openUrl).toHaveBeenNthCalledWith(1, env.CGU_LINK, undefined, true)
   })
 })
+
+function renderSuspensionChoice() {
+  return render(<SuspensionChoice />, {
+    // eslint-disable-next-line local-rules/no-react-query-provider-hoc
+    wrapper: ({ children }) => reactQueryProviderHOC(children),
+  })
+}
+
+function simulateSuspendForSuspiciousLoginSuccess() {
+  server.use(
+    rest.post(
+      env.API_BASE_URL + '/native/v1/account/suspend_for_suspicious_login',
+      async (_, res, ctx) => res(ctx.status(200))
```

Comment:
Pour mocker la réponse qu'une seule fois pour le test concerné
```suggestion
      async (_, res, ctx) => res.once(ctx.status(200))
```